pkg/cloudprovider/aws/object_storage_adapter.go

```diff
@@ -158,6 +170,7 @@ func (op *objectStorageAdapter) DeleteObject(bucket string, key string) error {
 }
 
 func (op *objectStorageAdapter) CreateSignedURL(bucket, key string, ttl time.Duration) (string, error) {
+	op.logger.Debug("create signed s3 url s3://%s/%s %s ttl", bucket, key, ttl)
```

Comment:
I'm curious what made you decide to make this debug vs info like the others?

Reply:
I think these make sense to all be at debug level -WDYT?